src/complete/237.delete-node-in-a-linked-list.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * @lc app=leetcode id=237 lang=typescript
+ *
+ * [237] Delete Node in a Linked List
+ */
+
+// Definition for singly-linked list.
+class ListNode {
+  val: number
+  next: ListNode | null
+  constructor(val?: number, next?: ListNode | null) {
+    this.val = (val === undefined ? 0 : val)
```

Comment:
Does the null coalescing operator work for this?
```suggestion
    this.val = val ?? 0
```
Same thing on the next line.

Reply:
The code for this was autogenerated by the initial problem code btw. Maybe they wanted to be explicit for some reason. Ya that’s what I would probably do if I wrote it. I think so.